src/discord.ts

```diff
@@ -87,6 +94,83 @@ discord.on(
   },
 );
 
+discord.on('message', async (message: Message) => {
```

Comment:
```suggestion
discord.on("message", async (message: Message) => {
```

Generally the formatting seems a bit off here, not blaming you here, but rather that the repo is missing prettier or an `.editorconfig` locally and simply a little diverging IDE settings. :D 

You could do `npm --save-dev prettier eslint-config-prettier eslint-plugin-prettier` for that whole formatting thing too, so it is not an issue in subsequent PRs at all. 

The repo for The Claw team site has a `.prettierrc` you can copy for the settings. -> https://github.com/whitep4nth3r/theclaw.team/blob/main/.prettierrc.json and for it to play nice with ESLint https://github.com/whitep4nth3r/theclaw.team/blob/main/.eslintrc (minus the next.js parts obviously, which are unnecessary in this case).